src/mockServer/mockServer.ts

```diff
@@ -1,6 +1,6 @@
 import { v4 } from 'uuid';
 import { createServer, Model, Factory, RestSerializer, Response, Request } from 'miragejs';
-import { API_ROUTES, buildUploadAppDataFilesRoute } from '../api/routes';
+import { API_ROUTES, buildPostChatBotRoute, buildUploadAppDataFilesRoute } from '../api/routes';
```

Comment:
You can de-structure `buildPostChatbotRoute` from the `API_ROUTES` object below on line 21 for example.